pjsip/src/pjsip/sip_transaction.c

```diff
@@ -139,6 +139,7 @@ static int max_retrans_count = -1;
 #define TIMEOUT_TIMER           2
 #define TRANSPORT_ERR_TIMER     3
 #define TRANSPORT_DISC_TIMER    4
+#define TERMINATE_TIMER         PJSIP_TSX_STATE_TERMINATED
```

Comment:
Not sure about this, `PJSIP_TSX_STATE_TERMINATED` has implicitly defined value, IMO better use `5` here?